_data/changelogs/side-navigation.yml

```
@@ -19,4 +19,49 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.1.0 release
+- date: 2022-08-05
+  summary: |-
+    Non-form buttons inside of usa-sidenav require `type="button"` to prevent default `submit` behaviors.
+  summaryAdditional: |-
+    This allowed us to remove `preventDefault()` to relevant components.
+  isBreaking: true
+  affectsAccessibility: true
+  affectsAssets: false
+  affectsGuidance: true
+  affectsJavascript: true
+  affectsMarkup: true
+  affectsStyles: false
+  githubPr: 4695
+  githubRepo: uswds
+  versionUswds: 3.1.0
+  # 3.0 release
+- date: 2022-04-28
+  summary: USWDS 3.0.0 released.
+  summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
+  isBreaking: false
+  affectsAccessibility: true
+  affectsAssets: true
+  affectsGuidance: true
+  affectsJavascript: true
+  affectsMarkup: true
+  affectsStyles: true
+  githubPr: 4656
+  githubRepo: uswds
+  versionUswds: 3.0.0
```

I made a couple small updates to the 3.0 release item. I updated the summary copy structure and identified it as a breaking change. Would you be able to update throughout? 

```suggestion
- date: 2022-04-28
  summary: Released USWDS 3.0.
  summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
  isBreaking: true
  affectsAccessibility: true
  affectsAssets: true
  affectsGuidance: true
  affectsJavascript: true
  affectsMarkup: true
  affectsStyles: true
  githubPr: 4656
  githubRepo: uswds
  versionUswds: 3.0.0
```

---

Committed suggestion and updated other group 3 components in c50e9b54960f6e75586bdf926e1c3b971143eaaa.